src/workloads/query/QueryStats.yml

```diff
@@ -46,6 +46,9 @@ ActorTemplates:
               pipeline: [
                 {
                   $queryStats: {^Parameter: {Name: "queryStatsParameters", Default: {}}}
+                },
+                {
+                  $count: "numDocuments"
```

Comment:
Can you please add a comment here indicating that the purpose of this is simply to ensure we've exhausted the cursor/data stream and looked at each partition?